Source/WebInspectorUI/UserInterface/Controllers/DebuggerManager.js

```diff
@@ -331,8 +330,10 @@ WI.DebuggerManager = class DebuggerManager extends WI.Object
             return WI.DebuggerManager.PauseReason.Timeout;
         case InspectorBackend.Enum.Debugger.PausedReason.Timer:
             return WI.DebuggerManager.PauseReason.Timer;
-        case InspectorBackend.Enum.Debugger.PausedReason.XHR:
-            return WI.DebuggerManager.PauseReason.XHR;
+        case InspectorBackend.Enum.Debugger.PausedReason.URL:
+        case InspectorBackend.Enum.Debugger.PausedReason.Fetch: // COMPATIBILITY (macOS 13.0, iOS 16.0): Debugger.paused.reason.Fetch was replaced by Debugger.paused.reason.URL
+        case InspectorBackend.Enum.Debugger.PausedReason.XHR: // COMPATIBILITY (macOS 13.0, iOS 16.0): Debugger.paused.reason.XHR was replaced by Debugger.paused.reason.URL
```

Comment:
Unless I'm mistaken these will effectively be `case undefined:` when the new protocol doesn't include them. While probably not a problem, it is a bit strange to think about.

Reply:
yeah it's maybe a bit odd, but this is definitely not the first time we've done this 😅